airmozilla/main/views.py

```diff
@@ -340,7 +340,12 @@ def get(self, request, slug):
             return self.cant_view_event(event, request)
 
         warning = None
-        if event.status not in (Event.STATUS_SCHEDULED, Event.STATUS_PENDING):
+        ok_statuses = (
+            Event.STATUS_SCHEDULED,
+            Event.STATUS_PENDING,
+            Event.STATUS_PROCESSING
```

Comment:
Always end with a comma. This is not true for JSON or for some Javascript linters. 
The reason you want to always end with a comma is that when you later add 1 more item to that tuple, it only becomes one line of change. If you also have to add a new comma to the previous line, it becomes two changed lines. 

Also, by always ending on a comma, when you some day do add another item there's a slower chance of accidentally causing a syntaxerror. 

One place where this rule is not true is when defining parameters to a function. E.g 
```python
def my_amazing_python_function(
    self, 
    argument_number_1,
    argument_number_2,
    argument_number_3
):
    # here is the body of the code
```